src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -68,8 +68,15 @@ public static IEnumerable<string> FindMissingOrChangedSourceFiles(this PdbFile p
 
         private static bool AreEqualBuffers(byte[] first, byte[] second)
         {
-            Argument.IsNotNull(nameof(first), first);
-            Argument.IsNotNull(nameof(second), second);
+            if (first == null && second == null)
```

Comment:
A null check vs. a check of a boolean local variable all compiles to the same assembly anyway. :)